buildSrc/src/main/kotlin/io/spine/internal/gradle/Repositories.kt

```diff
@@ -110,6 +115,44 @@ object PublishingRepos {
         credentialsFile = "cloudrepo.properties"
     )
 
+    /**
+     * The experimental Google Cloud Artifact Registry repository.
+     *
+     * In order to successfully publish into this repository, a service account key is needed.
+     * The published must create a service account, grant it the permission to write into
+     * Artifact Registry, and generate a JSON key.
+     * Then, the key must be placed somewhere on the file system and the environment variable
+     * `GOOGLE_APPLICATION_CREDENTIALS` must be set to point at the key file.
+     * Once these preconditions are met, publishing becomes possible.
+     *
+     * ## Implementation note
+     * Google provides [tools](https://github.com/GoogleCloudPlatform/artifact-registry-maven-tools)
+     * for configuring authentication for the Maven repositories, including a Gradle plugin.
+     * However, the plugin is incompatible with Gradle 7.x at the moment.
+     * For now, we reproduce what the plugin does manually. This makes the whole `buildSrc`
+     * depend on the `artifactregistry-auth-common` artifact. Track [this issue](https://github.com/GoogleCloudPlatform/artifact-registry-maven-tools/issues/52)
```

Comment:
Please have `Track this issue...`  coming as a next paragraph. And “‘please‘ would be nice”.